diff --git a/config.h b/config.h
index cd89b5f..137a08e 100644
--- a/config.h
+++ b/config.h
@@ -135,6 +135,8 @@ static const keymap_t keys[] = {
 			"jpegtran -rotate 270 -copy all -outfile \"$SXIV_IMG\" \"$SXIV_IMG\"" },
 	{ true,   XK_period,        it_shell_cmd,         (arg_t) \
 			"jpegtran -rotate  90 -copy all -outfile \"$SXIV_IMG\" \"$SXIV_IMG\"" },
+    { true,   XK_Return,        it_shell_cmd,         (arg_t)
+            "echo \"$SXIV_IMG\""},
 };
 
 /* mouse button mappings for image mode: */
