diff --git a/config.h b/config.h
index f5d048a..991d73c 100644
--- a/config.h
+++ b/config.h
@@ -157,6 +157,8 @@ static const keymap_t keys[] = {
 			"jpegtran -rotate  90 -copy all -outfile \"$SXIV_IMG\" \"$SXIV_IMG\"" },
 	{ true,   XK_slash,         it_shell_cmd,         (arg_t) \
 			"jpegtran -rotate 180 -copy all -outfile \"$SXIV_IMG\" \"$SXIV_IMG\"" },
+
+	{ true,   XK_Return,        it_toggle_image_mark, (arg_t) None },
 };
 
 /* mouse button mappings for image mode: */
